packages/lab/src/banner/Banner.css

```diff
@@ -56,12 +57,6 @@
   color: var(--uitkBanner-icon-color, var(--banner-status-icon-color));
   fill: var(--uitkBanner-icon-color, var(--banner-status-icon-color));
   margin-right: var(--uitkTooltip-icon-marginRight, 6px);
-  padding-top: var(--banner-icon-top);
```

Comment:
should delete --banner-icon-top if no longer needed

Reply:
Good spot, removed